src/Microsoft.Diagnostics.Runtime/src/DataTargets/DataTarget.cs

```diff
@@ -15,9 +15,6 @@
 using System.Linq;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-#if NET5_0
```

Comment:
It's fine to strip this.  The real reason I left it in is because it's _supposed_ to work, and would work if I do the versioning fix and pick up a new copy of arcade.  I planned to do that work and had other things going on...